src/h2load.cc

```diff
@@ -963,6 +965,31 @@ std::vector<std::string> read_uri_from_file(std::istream &infile) {
 } // namespace
 
 namespace {
+// Called every second when rate mode is being used
+void second_timeout_cb(EV_P_ ev_timer *w, int revents) {
+  auto config = static_cast<Config *>(w->data);
+
+  auto nclients_per_worker = config->rate;
+  auto nreqs_per_worker = config->max_concurrent_streams * config->rate;
+
+  if (config->current_worker >= std::max((ssize_t)0, (config->seconds - 1))) {
+    nclients_per_worker = config->rate + config->conns_remainder;
+    nreqs_per_worker = (int)config->max_concurrent_streams *
+                       (config->rate + config->conns_remainder);
```

Comment:
I think we should assign 0 to config->conns_remainder here, without that, we keep adding remainder to all subsequent worker, which increases connections and requests.